datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -621,9 +623,12 @@ async fn roundtrip_expr_api() -> Result<()> {
             lit(1),
         ),
         array_replace_all(make_array(vec![lit(1), lit(2), lit(3)]), lit(2), lit(4)),
-        first_value(vec![lit(1)], false, None, None, None),
+        first_value(lit(1), Some(vec![lit(2)])),
+        first_value_builder(lit(1)).order_by(vec![lit(3)]).build(),
         covar_samp(lit(1.5), lit(2.2)),
+        covar_samp_builder(lit(1.5), lit(2.3)).build(),
         covar_pop(lit(1.5), lit(2.2)),
+        covar_pop_builder(lit(1.5), lit(2.3)).build(),
```

Comment:
I think roundtrip tests also cover the testing for these expression APIs.

I expect we add all the functions here for test, and only little functions in `datafusion-example`.